src/training/nn_trainer.py

```diff
@@ -93,3 +99,47 @@ def save_images(self, batch_size, input_var, iteration, loader, path_fake, path_
             generated_output = gen(z)
             self.dataloader.save_images(generated_output, shape, path_fake)
             gen.train()
+
+    def save_checkpoint(self, generators, discriminators, cell_number):
+        checkpoint = dict()
+
+        checkpoint['time'] = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+
+        if len(generators) > 0:
```

Comment:
Is generator and discriminator code different in anyway except the prefix? If not it could be made into a function that takes the prefix as an argument to reduce code duplication

Reply:
Sure!!